hive/src/main/java/io/confluent/connect/storage/hive/HiveConfig.java

```diff
@@ -175,6 +194,37 @@
           schemaCompatibilityRecommender
       );
     }
+
+    {
+      // Define Logical type handling configuration group
+      final String group = "Logical type handling";
+      int orderInGroup = 0;
+
+      // Define Logical type handling configuration group
+      CONFIG_DEF.define(
+          HIVE_LOGICAL_TYPES_ENABLE_CONFIG,
+          Type.BOOLEAN,
+          HIVE_LOGICAL_TYPES_ENABLE_DEFAULT,
+          Importance.MEDIUM,
+          HIVE_LOGICAL_TYPES_ENABLE_DOC,
+          group,
+          ++orderInGroup,
+          Width.SHORT,
+          HIVE_LOGICAL_TYPES_ENABLE_DISPLAY
+      );
+
+      CONFIG_DEF.define(
+          DECIMAL_PRECISION_CAP_CONFIG,
+          Type.INT,
+          DECIMAL_PRECISION_CAP_DEFAULT,
+          Importance.MEDIUM,
```

Comment:
as mentioned above, a validator and a recommender might be a good option here. 
Feel free to create a single inner static class that implements both interfaces.